src/Compilers/CSharp/Portable/Syntax/StackAllocArrayCreationExpressionSyntax.cs

```diff
@@ -11,6 +11,14 @@ public StackAllocArrayCreationExpressionSyntax Update(SyntaxToken stackAllocKeyw
             return Update(StackAllocKeyword, type, default(InitializerExpressionSyntax));
         }
     }
+
+    public partial class LocalDeclarationStatementSyntax
+    {
+        public LocalDeclarationStatementSyntax Update(SyntaxTokenList modifiers, VariableDeclarationSyntax declaration, SyntaxToken semicolonToken)
+        {
+            return Update(default(SyntaxToken), modifiers, declaration, semicolonToken);
```

Comment:
you should be able to just use 'default' instead of 'default(SyntaxToken)' right?